projects/RabbitMQ.Client/client/impl/WireFormatting.Write.cs

```diff
@@ -59,7 +59,7 @@ public static int WriteArray(ref byte destination, IList val)
             return bytesWritten;
         }
 
-        public static int GetArrayByteCount(IList val)
+        public static int GetArrayByteCount(IList? val)
```

Comment:
There is no usage of this method where null can be passed. Consider to change `List? val` to `List val` and remove unnecessary null check in this method.

Reply:
WireFormatting is formatting according to the protocol, and the list can be null, so even if it isn't used, I wouldn't change it.